examples/recording_service/service_admin/cpp/README.md

```diff
@@ -142,6 +156,34 @@ parameters:
     requires a name parameter and optionally, a description (remember to use
     quotes for multi-word descriptions).
 
+-   [Optional] `--add-breakpoint`: this command-line parameter allows the user
+    to define a breakpoint for *Replay Service*. These breakpoint will be used
+    in the debug mode of Replay. This command requires a timestamp in nanosecond
+    form and optionally, a breakpoint name.
+
+-   [Optional] `--remove-breakpoint`: it will allow the user to remove an
+    existed breakpoint of the replay. This command requires the timestamp or
+    name of the breakpoint to be removed.
+
+-   [Optional] `--goto-breakpoint`: this command-line parameter the user to jump
+    to a existed breakpoint. This command requires the timestamp or name of the
+    specific breakpoint.
+
+-   [Optional] `--continue-seconds`: this command-line parameter allows the user
+    to continue the replay for a specific amount of seconds. This command
+    requires the number of seconds to be replayed.
+
+-   [Optional] `--continue-slices`: it will allow the user to continue the
+    replay for a specific amount of slices. This command requires the number of
+    slices to be replayed.
+
+-   [Optional] `--current-timestamp`: it will allow the user to jump in time
+    forward and backward in *Replay Service*. This command requires a timestamp
+    in nanosecond format.
+
+Note: In order to run the *Replay Service* debug mode commands you should
```

Comment:
is it "should" or "must"?

Reply:
I think "should" because they can also use the command line option -debugMode